### ISSUE TITLE
Wrong colors

### ISSUE THREAD

**Issue by radpopl** [v0lt/ScriptSourceFilter#1]
Same effect in MPC-BE and MPC-HE.

With internal filter everything is OK. Also in VDub.

[broken link removed]

**Comment by v0lt** [comment]
I didn't understand anything.

Please specify the version of AviSynth or VapourSynth.
Please provide a working script and other necessary files.
Please make normal screenshots that are visible.

**Comment by radpopl** [comment]
<img width="376" alt="screenshot" src="https://github.com/user-attachments/assets/612db096-6640-4b08-a440-4292d9c0db63">

Latest Avisynth, MPC Source Filter.
The script is just ffms2("file.mp4"). Blue and red swapped?

In MPC-HC the same thing happens with this script.

**Comment by v0lt** [comment]
Windows 10 x64 22H2
AviSynth+ 3.7.3 (r4003, 3.7, x86_64)
FFmpegSource (FFMS2) 5.0 in "plugins64+" folder 

MPC-BE x64 1.8.2.
MPC Script Source 0.2.4.178
MPC Video Renderer 0.8.9.2318

MediaInfo report for video stream:
```
Video
ID                             : 224 (0xE0)
Format                         : MPEG Video
Format version                 : Version 2
Format profile                 : Main@Main
Format settings                : CustomMatrix / BVOP
Format settings, BVOP          : Yes
Format settings, Matrix        : Custom
Format settings, GOP           : M=3, N=12
Format settings, picture struc : Frame
Duration                       : 2 min 52 s
Bit rate mode                  : Variable
Bit rate                       : 5 369 kb/s
Maximum bit rate               : 15.0 Mb/s
Width                          : 720 pixels
Height                         : 576 pixels
Display aspect ratio           : 16:9
Frame rate                     : 25.000 FPS
Standard                       : PAL
Color space                    : YUV
Chroma subsampling             : 4:2:0
Bit depth                      : 8 bits
Scan type                      : Interlaced
Scan order                     : Top Field First
Compression mode               : Lossy
Bits/(Pixel*Frame)             : 0.518
Time code of first frame       : 00:00:00:00
Time code source               : Group of pictures header
GOP, Open/Closed               : Open
GOP, Open/Closed of first fram : Closed
Stream size                    : 110 MiB (94%)
```

Script content:
```
FFmpegSource2("C:\Temp\clip.mpg")
propShow()
```
I get the correct result in terms of colors and proportions.
![clipAR](https://github.com/user-attachments/assets/526c43cd-75ef-4cde-b0b5-946ff2cb02f1)
I tried on other video files with proportions and always get the correct image.

List of active filters
![filterList](https://github.com/user-attachments/assets/a0fdcae0-13b9-41ce-98ae-2427da9b0f27)

**Comment by v0lt** [comment]
You didn't report anything other than the screenshot and didn't provide what I asked for. I'm no longer interested in asking you for information. You have something local, figure it out yourself.

**Comment by radpopl** [comment]
Internal filter (same with and all other programs - VDub, ffmpeg):
<img width="348" alt="int" src="https://github.com/user-attachments/assets/7ae6d9be-6401-4f1e-a985-d08bd2737a6c">
MPC Script Source (MPC-BE, MPC-HC)
<img width="380" alt="ext" src="https://github.com/user-attachments/assets/424a6aa7-a1a5-467b-9015-037602efdb4d">
![image](https://github.com/user-attachments/assets/d604a2f1-6dd8-4511-9680-3bf3850ea8a4)

Do you really think this has anything to do with the AviSynth version or the script?
It will probably be with the graphics driver, as I've already heard about the problem of swapping red and blue in some programs (some Adrenaline drivers months ago).

**Comment by v0lt** [comment]
Provide your video file and script.

**Comment by radpopl** [comment]
[a.zip](https://github.com/user-attachments/files/18083402/a.zip)

Script shortened to few lines. convertbits(10) causes the problem.

**Comment by v0lt** [comment]
Fixed in commit 21ebc2e.

Test build 0.2.4.180 - https://disk.yandex.ru/d/ulwdvtB-75Rhwg
